README.md

```diff
@@ -1,31 +1,43 @@
+
+This and other `ap_verify` "datasets" are based on [ap_verify_dataset_template](https://github.com/lsst-dm/ap_verify_dataset_template).
 
 Contains DECam data from the HiTS (2015) fields `Blind15A_26`, `Blind15A_40`, and `Blind15A_42`.
 
+---
+**Note**
+This repo contains over 80 DECam images with 60+ CCDs each! If you want a smaller dataset to practice running `ap_verify`, consider `ap_verify_ci_hits2015` or `ap_verify_ci_cosmos_pdr2`.
+
+At present, this repository can only be used for Gen 2 processing.
+Support for the new Gen 3 pipeline framework is expected in the future.
+
+---
+
 Relevant Files and Directories
 -----
 path                  | description
 :---------------------|:-----------------------------
 `raw`                 | Raw, compressed DECam fits images from the HiTS (2015) fields `Blind15A_26`, `Blind15A_40`, and `Blind15A_42`.
-`calib`               | DECam master calibs and `wtmap`s from the 2015 HiTS campaign. No raw images. See below for filename information.
-`config`              | Dataset-specific configs to help Stack code work with this dataset.
-`templates`           | Butler repo containing coadded images intended to be used as templates.
-`repo`                | A template for a Butler raw data repository. This directory must never be written to; instead, it should be copied to a separate location, and data ingested into the copy (this is handled automatically by `ap_verify`, see below). Currently contains the appropriate DECam `_mapper` file.
+`calib`               | DECam Community Pipeline MasterCalibs and `wtmap`s from the 2015 HiTS campaign. No raw images. See below for filename information.
+`config`              | Dataset-specific configs to help the Science Pipelines work with this dataset.
+`templates`           | Gen2 Butler repo containing good seeing g-band coadds and a skymap. For use as difference imaging templates.
+`repo`                | A template for a Gen2 Butler raw data repository. This directory must never be written to; instead, it should be copied to a separate location, and data ingested into the copy (this is handled automatically by `ap_verify`, see below). Currently contains the appropriate DECam `_mapper` file.
 `refcats`             | Tarballs of Gaia and PS1 reference catalogs in HTM format for regions overlapping all three HiTS fields.
 `dataIds.list`        | List of dataIds for use in running Tasks. Currently set to run all Ids.
+`preloaded`           | Preliminary Gen3 Butler repo containing a skymap as well as the refcats, calibs, and templates described above.
 
 Master calibration file names
 -----------------------------
 
 Below are the different types of master calibration files in the `calib` directory:
 
 * fci: dome flat images
-* fcw: dome flat weight maps
+* fcw: dome flat weight maps (not used)
 * zci: zero/bias images
-* zcw: zero/bias weight maps
-* ici: illumcor images
+* zcw: zero/bias weight maps (not used)
+* ici: illumcor images (not used)
```

Comment:
Isn't the "not used" mainly up to the pipeline, not the repository? I'm not sure I want to document something that's outside our control.

Reply:
We don't even ingest them, they literally aren't used by anything in our science pipelines code and should never have been committed in the first place 😬 (but I'm restraining myself from touching calib files on this ticket since that quickly became a mess)